plugins/catalog-import/src/components/DefaultImportPage/DefaultImportPage.tsx

```diff
@@ -47,14 +59,8 @@ export const DefaultImportPage = () => {
           </SupportButton>
         </ContentHeader>
 
-        <Grid container spacing={2} direction="row-reverse">
-          <Grid item xs={12} md={4} lg={6} xl={8}>
-            <ImportInfoCard />
-          </Grid>
-
-          <Grid item xs={12} md={8} lg={6} xl={4}>
-            <ImportStepper />
-          </Grid>
+        <Grid container spacing={2}>
```

Comment:
Not super fond of the array approach - could this one have had `direction={isMobile ? 'column-reverse' : 'row'}` and instead letting the content inside be fixed?

Reply:
The issue is direction prop is purely graphical, it does nothing for the logical order of the Html layout, meaning even if it looked like the stepper was at the left, the tab order would go through the InfoCard first, so here are the 3 options:

- Laying out the Html in the same order on desktop & mobile, with no condition for reverse or anything.
- Logical sorting of the components kin of like I did in this PR, a bit weird since it needs us to use `mediaQuery`.
- We label this as won't fix, ignoring it altogether.

Our Accessibility expert told us about the issue, my guess is it's because of the WCAG 2.0 recommendation, the question is if we consider the order of the keyboard navigation should match the UI render in this case: https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction#accessibility_concerns

Reply:
I see. Yeah I don't know enough to have a strong opinion there, and arguably the "cost" in terms of code here isn't all too bad. So let's go ahead and 🚢